frontend/src/scenes/organization/TeamMembers/BulkInviteModal.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button, Col, Input, Row } from 'antd'
+import Modal from 'antd/lib/modal/Modal'
+import { useActions, useValues } from 'kea'
+import React, { useEffect } from 'react'
+import { userLogic } from 'scenes/userLogic'
+import { PlusOutlined } from '@ant-design/icons'
+import './BulkInviteModal.scss'
+import { isEmail } from 'lib/utils'
+import { bulkInviteLogic } from './bulkInviteLogic'
+
+const PLACEHOLDER_NAMES = ['Jane', 'John']
+
+function InviteRow({ index }: { index: number }): JSX.Element {
+    const name = PLACEHOLDER_NAMES[index % 2]
+
+    const { invites } = useValues(bulkInviteLogic)
+    const { updateInviteAtIndex } = useActions(bulkInviteLogic)
+
+    return (
+        <Row gutter={16} className="invite-row">
+            <Col xs={12}>
+                <Input
+                    placeholder={`${name.toLowerCase()}@posthog.com`}
+                    type="email"
+                    className={`error-on-blur${!invites[index].isValid ? ' errored' : ''}`}
+                    onChange={(e) => {
+                        const { value } = e.target
+                        let isValid = true
+                        if (value && !isEmail(value)) {
+                            isValid = false
+                        }
+                        updateInviteAtIndex({ email: e.target.value, isValid }, index)
+                    }}
+                    value={invites[index].email}
+                />
+            </Col>
+            <Col xs={12}>
+                <Input
+                    placeholder={name}
+                    onBlur={(e) => {
```

Comment:
Using onBlur seems kind of weird here, not sure this would always work. Why not onChange?

Reply:
Oops forgot to change that, yes